docs/tendermint-core/configuration.md

```diff
@@ -457,12 +457,9 @@ timeout_prevote = "1s"
 timeout_prevote_delta = "500ms"
 timeout_precommit = "1s"
 timeout_precommit_delta = "500ms"
-timeout_commit = "1s"
+target_height_duration = "1s"
```

Comment:
[optional] these are just docs so the value doesn't matter but proposal for it to match the 15s on line 354

```suggestion
target_height_duration = "15s"
```